src/commands.py

```
@@ -0,0 +1,189 @@
+        :param model: The model identifier to set as default.
+        """
+        if not await self._check_admin(interaction):
+            return
+
+        config_manager.set_default_model(model)
+        channel_name = getattr(interaction.channel, "name", "DM")
+        logger.info("Admin %s switched default model to %s (command sent from #%s)", interaction.user.name, model, channel_name)
+        await interaction.response.send_message(f"[Default model set to `{model}`.]")
+
+    @config_model.autocomplete("model")
+    async def model_autocomplete(self, interaction: discord.Interaction, current: str) -> list[Choice[str]]:
```

Ruff (0.14.10)

Unused method argument: `interaction`

(ARG002)